content/en/synthetics/browser_tests/actions.md

```diff
@@ -93,9 +93,19 @@ Asserts that an email was sent and whether specific values (`string`, `number`,
 
 #### Test your UI with custom JavaScript
 
-Test a custom assertion on the active page using your own JavaScript scripts. JavaScript assertions support both synchronous and asynchronous code. Since JavaScript assertions run in the context of the active page, it also means these steps can access all the objects defined in the active page (libraries, builtins, global variables, ...).
+Test a custom assertion on the active page using your own JavaScript scripts. JavaScript assertions support both synchronous and asynchronous code. 
 
-If you need to load external libraries, you can wrap their loading in a promise like below:
+The JavaScript assertion function comes with two parameters. You also need to make sure to include a `return` statement in your JavaScript snippet.
+
+* `vars`: String containing your browser test [variables][5]. Use `vars.<YOUR_VARIABLE>` to refer to your browser test variable in your JavaScript snippet. Example: If your browser test contains a `USERNAME` variable, you can call it in your JavaScript snippet using `vars.USERNAME`.
+
+* `element`: Locator of the element on the page. This element can be selected and updated with the `Select` and `Update` target element buttons of the step. The selected element automatically leverages Datadog browser test multi locating algorithm.
+
+{{< img src="synthetics/browser_tests/js_assertion.mp4" alt="Browser Test JavaScript Assertion" video="true" width="100%">}}
+
+* The `return` statement needs to reflect the condition the assertion needs to meet for your browser test step to be successful. You can return any type, but the value is automatically casted as a boolean.
+
+Since JavaScript assertions run in the context of the active page, it also means these steps can access all the objects defined in the active page (libraries, builtins, global variables, ...). If you need to load external libraries, you can wrap their loading in a promise like below:
```

Comment:
```suggestion
Since JavaScript assertions run in the context of the active page, these steps can access all the objects defined in the active page (libraries, built-ins, global variables, etc.). To load external libraries, use a promise, for example:
```